HIP/hip-991.md

```diff
@@ -139,16 +123,15 @@ When creating a topic, the FEKL list is independent and must be explicitly popul
 
 ### HIP Parameters
 
-After discussing with the engineering team, the HIP parameters are defined as follow:
+The HIP parameters are defined as follow:
 
 * `MAX_CUSTOM_FEE_ENTRIES_FOR_TOPICS = 10`
 * `MAX_ENTRIES_FOR_FEE_EXEMPT_KEY_LIST = 10`
-* `MAX_FEE = unsigned int 64`
 
 ### User Flows and Interaction
 
 * Users will specify the fee settings during the topic creation process through a simple interface in their Hedera client (refer to the creation of token custom fees/fixed fee for reference).
-* Before submitting a message to a topic through an application or wallet interface, users must set an allowance and a maximum fee per message.
+* When submitting a message to a topic with custom fees through an application or wallet interface, users must set the maximum fee for the message.
```

Comment:
to accommodate the requirement of accepting all fees, should we add a line that is something like:
* Users submitting messages can add a flag to accept all custom fees from a topic id

Reply:
Added

Reply:
Do we want to specify what this flag is called?